### ISSUE TITLE
[MU4 Issue] VST. Plugin window can't be opened and user is switched to the main desktop when MU is in the full-screen mode (Mac only)

### ISSUE THREAD

**Issue by boryskuzmenko** [musescore/MuseScore#10946]
**To Reproduce**
1. Open any score in MU
2. Switch MU to full-screen mode
3. Apply a VSTi to any track
4. Try to open VST window

**Expected behavior**
VST window can be opened in any cases without switching between desktops

**Screenshots**

https://user-images.githubusercontent.com/101250347/160626707-99ac0f46-af29-42e3-aacb-603c8a1cb6e1.mp4

**Platform information**
 macOS 12.3

**Additional context**
The issue is reproducible for all my instruments except Kontakt Player (IK Sample tank, discoDSP OB-xd, Spitfare LAB, Vital)

**Comment by Tantacrul** [comment]
I'm not getting this issue on my Mac any more. How about you @boryskuzmenko?

If you are, please reopen this issue.

**Comment by boryskuzmenko** [comment]
Yes, I can reproduce this bug on all plugins except Native Instruments ones

https://user-images.githubusercontent.com/101250347/171226264-3e5a0757-1d85-4ebf-8ed2-c08f6d29eb58.mov

**Comment by Tantacrul** [comment]
Ah. Native Instruments was the one VST I tested. Lol. Thanks!

**Comment by cbjeukendrup** [comment]
This seems fixed in 4.6
